fmn/fasjson_client.py

```diff
@@ -20,6 +21,7 @@ def __init__(self, url, principal=None):
         self.url = url
         self.principal = principal
         creds = None
+        os.environ["KRB5_CLIENT_KTNAME"] = "/etc/krb5.keytab"
```

Comment:
That should probably be set on the system in `/etc/krb5.conf.d`, in case sysadmins want to put the keytab in a different place (that's common when dealing with openshift pods for example, because configs are mounted). So I think this change should be in Ansible.

Reply:
thanks, addressed now. i'll make a seperate pr for that on ansible